src/components/Publish/_validation.ts

```diff
@@ -131,7 +131,8 @@ export const validationSchema: Yup.SchemaOf<any> = Yup.object().shape({
   user: Yup.object().shape({
     stepCurrent: Yup.number(),
     chainId: Yup.number().required('Required'),
-    accountId: Yup.string().required('Required')
+    accountId: Yup.string().required('Required'),
+    hasBalance: Yup.bool().isTrue('Insufficient balance.').required('Required')
```

Comment:
so which error message is shown then? Cause in `Publish/Pricing/Dynamic.tsx` we somehow manage our own error state in there instead of using the actual form errors so based on that it seems this message here would never be shown? And which one should be shown?

Reply:
yeah, there's no need to show anything. We just need to make sure is `true`